polaris:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  front_left_steering_position_controller:
    type: effort_controllers/JointPositionController
    joint: polaris_ranger_ev__front_left_steering_joint
    pid: {p: 1000.0, i: 100.0, d: 10.0}

  front_right_steering_position_controller:
    type: effort_controllers/JointPositionController
    joint: polaris_ranger_ev__front_right_steering_joint
    pid: {p: 1000.0, i: 100.0, d: 10.0}

  # Effort Controllers -----------------------------------------
  front_left_wheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: polaris_ranger_ev__front_left_wheel_joint
    pid: {p: 1000.0, i: 100.0, d: 0.1}

  front_right_wheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: polaris_ranger_ev__front_right_wheel_joint
    pid: {p: 1000.0, i: 100.0, d: 0.1}

  rear_left_wheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: polaris_ranger_ev__rear_left_wheel_joint
    pid: {p: 1000.0, i: 100.0, d: 0.1}

  rear_right_wheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: polaris_ranger_ev__rear_right_wheel_joint
    pid: {p: 1000.0, i: 100.0, d: 0.1}

